django/applications/catmaid/control/cropping.py

```diff
@@ -496,14 +480,16 @@ def extract_substack_no_rotation(job) -> List:
                                      (estimated_total_size,
                                       settings.GENERATED_FILES_MAXIMUM_SIZE))
                 # Draw the image onto result image
-                cropped_slice.composite( image, ip.x_dst, ip.y_dst, co.OverCompositeOp )
+                cropped_slice.paste(image, ip.x_dst, ip.y_dst)
                 # Delete tile image - it's not needed anymore
                 del image
 
-            if cropped_slice:
+            if cropped_slice:  # TODO: always true?
```

Comment:
Attention needed.

Reply:
Good catch. To me it looks like this conditional can be removed and the body can be executed all the time (which is, like your comment says, what is happening anyway).

Reply:
Since I merged this branch already, I applied this change and removed the condition.